pkg/server/api/agent/v1/service.go

```diff
@@ -387,6 +387,20 @@ func (s *Service) RenewAgent(ctx context.Context, req *agentv1.RenewAgentRequest
 		return nil, api.MakeErr(log, codes.Internal, "caller ID missing from request context", nil)
 	}
 
+	attestedNode, err := s.ds.FetchAttestedNode(ctx, callerID.String())
+	if err != nil {
+		return nil, api.MakeErr(log, codes.Internal, "failed to fetch agent", err)
+	}
+
+	if attestedNode == nil {
+		return nil, api.MakeErr(log, codes.NotFound, "agent not found", err)
+	}
+
+	// Log warning for now, change to return PermissionDeniedDetails_AGENT_MUST_REATTEST in a future release.
+	if attestedNode.CanReattest {
+		log.Warn("Renewing SVID for agent that should be reattesting. In the next release this will result in an error to the agent.")
```

Comment:
mmm I'm not sure if warning for this is something we must do, maybe a debug? 
we'll populating warnings for something that users does not have control at all

Reply:
Yeah, I'm not sure about this log line either....

Reply:
since we are using the feature flag we don't need the log line anymore. It just return error if the feature flag is enabled.